packages/server/routes/personality.js

```diff
@@ -10,27 +11,67 @@ const router = require('../lib/util').router();
 /**
  * GET {domain}/personality
  */
-router.get('/', personality.listAll);
+router.get('/', listAll);
```

Comment:
no need to create the function listAll, just pass it as a callback. Also, use arrow functions.